src/components/RecordingDialog.tsx

```diff
@@ -56,6 +56,9 @@ const RecordingDialog = ({
   );
   const countdownStages: CountdownStage[] = useMemo(
     () => [
+      // Empty string state added to countdown because aria-live role region
+      // reads dynamic changes to region and not the initial contents.
+      { value: "", duration: 500, fontSize: "8xl" },
```

Comment:
Not sure if this is the right way of doing it, suggestions welcomed.

Alternatively, we could use `role="alert"` instead of `role="region"`, but mdn documents role alert as for errors/warning messages (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes).

Reply:
It's odd that the dialog starts empty. Can we have an empty string for the minimum amount of time necessary rather than 500 ms for the change to be read out?

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/370/commits/0b9dc4b8da1b1a8eff9433ea092bd62e37e88f39